README.md

```diff
@@ -86,31 +140,45 @@ Each directory in this repo is a sample Temporal project built with the [TypeScr
 - **Next.js**:
   - [**One-click e-commerce**](https://github.com/temporalio/samples-typescript/tree/main/nextjs-ecommerce-oneclick): Buy an item with one click, and the Workflow will wait 5 seconds to see if the user cancels before it executes the order.
+- **Express**:
+  - [`vkarpov15/temporal-ecommerce-ts`](https://github.com/vkarpov15/temporal-ecommerce-ts): The `cartWorkflow` used in [this blog series](https://docs.temporal.io/blog/build-an-ecommerce-app-with-temporal-part-1/)
+  - [`temporal-rest`](https://www.npmjs.com/package/temporal-rest): Express middleware router that automatically exposes endpoints for Workflows, Signals, and Queries.
+- **Remix**:
+  - [`gustavofsantos/temporal-survey`](https://github.com/gustavofsantos/temporal-survey)
+- **NestJS**:
+  - [`nestjs-temporal`](https://www.npmjs.com/package/nestjs-temporal)
+
+### Other
+
+- **Chatbots**:
+  - [`JoshuaKGoldberg/temporal-adventure-bot`](https://github.com/JoshuaKGoldberg/temporal-adventure-bot): Choose-your-own-adventure Slack/Discord chatbot (see [tutorial](https://docs.temporal.io/typescript/chatbot-tutorial) and [video](https://www.youtube.com/watch?v=hGIhc6m2keQ))
+- **Caching**:
+  - [`vkarpov15/temporal-api-caching-example`](https://github.com/vkarpov15/temporal-api-caching-example/): Cache data from a third-party API (see [blog post](https://docs.temporal.io/blog/caching-api-requests-with-long-lived-workflows))
+- **DSL Control Flows**:
+  - [**YAML DSL Interpreter**](https://github.com/temporalio/samples-typescript/tree/main/dsl-interpreter): Make Workflows interpret a custom YAML-based Domain Specific Language of your design.
+  - [**XState Interpreter**](https://github.com/Devessier/temporal-electronic-signature): Interpret XState state charts in a Workflow. Presented at [the November 2021 meetup](https://youtu.be/GpbOkDjpeYU?t=1616).
+- **URL Scraping**
+  - [`andreasasprou/temporal-url-batch-scraping`](https://github.com/andreasasprou/temporal-url-batch-scraping)
 - **GraphQL**:
-  - [**Temporal + GraphQL**](https://github.com/vkarpov15/temporal-graphql): Expose Temporal Workflows in an Apollo GraphQL API
-- **Example Apps**:
-  - Subscription Workflow example: https://github.com/temporalio/subscription-workflow-project-template-typescript/ (useful to compare with other SDKs)
-  - E-commerce example: https://github.com/vkarpov15/temporal-ecommerce-ts
-  - Chatbot example with Slack/Discord: [Walkthrough](https://docs.temporal.io/typescript/chatbot-tutorial), [Video](https://www.youtube.com/watch?v=hGIhc6m2keQ), [Code](https://github.com/JoshuaKGoldberg/temporal-adventure-bot)
-  - Temporal + Remix Survey app: https://github.com/gustavofsantos/temporal-survey
-  - URL Batch Scraping demo: https://github.com/andreasasprou/temporal-url-batch-scraping
+  - [`vkarpov15/temporal-graphql`](https://github.com/vkarpov15/temporal-graphql): Generate a GraphQL schema from Workflow code (see also [`temporalio/graphql`](https://github.com/temporalio/graphql))
```

Comment:
I would recommend removing this link. temporal-graphql is unfortunately not viable in its current form. No good way to get static typing info at runtime from Temporal to GraphQL currently.